src/bundle/Resources/views/themes/admin/ui/search_tag.html.twig

```diff
@@ -0,0 +1,19 @@
+{% set btnClass = btnClass|default('') %}
+{% set content = content|default('') %}
+{% set targetSelector = targetSelector|default('') %}
+{% set title = title|default('') %}
+
+<div class="ez-tag ez-search-criteria-tags__tag" {% if title %} title="{{ title }} {% endif %}">
+    <div class="ez-tag__content">
+        {{ content }}
+    </div>
+    <button 
+        type="button" 
+        class="ez-tag__remove-btn {{ btnClass }}"
+        data-target-selector="{{ targetSelector }}"
+    >
+        <svg class="ez-icon ez-icon--medium ez-icon--dark">
+            <use xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#circle-close"></use>
```

Comment:
asset()

Reply:
@lucasOsti I've reported 2 issues regarding this PR on task comment in jira.